nvflare/app_common/utils/fl_model_utils.py

```diff
@@ -193,3 +193,18 @@ def set_meta_prop(model: FLModel, key: str, value: Any):
     @staticmethod
     def get_configs(model: FLModel) -> Optional[dict]:
         return FLModelUtils.get_meta_prop(model, MetaKey.CONFIGS)
+
+    @staticmethod
+    def update_model(model: FLModel, model_update: FLModel, replace_meta: bool = True) -> FLModel:
+        if replace_meta:
+            model.meta = model_update.meta
+        else:
+            model.meta.update(model_update.meta)
+        if model_update.params_type == ParamsType.FULL:
+            model.params = model_update.params
+        elif model_update.params_type == ParamsType.DIFF:
+            for v_name, v_value in model_update.params.items():
+                model.params[v_name] = model.params[v_name] + v_value
```

Comment:
You might want to also check the `model`'s params_type, what if it has ParamsType.DIFF?
What do we want in that case, raise an RuntimeError as well?

Reply:
Good point. I added that check:
```
        if model.params_type != ParamsType.FULL:
            raise RuntimeError(f"params_type {model_update.params_type} `model` not supported! Expected `ParamsType.FULL`.")
```